tests/ssl-opt.sh

```diff
@@ -4302,6 +4304,7 @@ run_test    "Authentication, CA callback: server ECDH p256v1, client optional, p
             -c "! Certificate verification flags"\
             -c "bad server certificate (ECDH curve)" # Expect failure only at ECDH params check
 
+requires_config_enabled MBEDTLS_SHA384_C
```

Comment:
I believe this one was failing for some reason, although I cannot recall what it was. I think adding this resolved the issue, but I'll remove it now and verify if it still occurs, it's possible it's not really necessery. Thanks for spotting this.

Reply:
There's already a mechanism to skip test cases that require a ciphersuite if that ciphersuite is not enabled. See `maybe_requires_ciphersuite_enabled`. So adding explicit dependencies just because of the ciphersuite should not be required.